core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1878,4 +1878,34 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertTrue("Expected consumer2 to consume one message from offset 1, which is the committed offset of consumer1",
       records2.count() == 1 && records2.records(tp).asScala.head.offset == 1)
   }
+
+  @Test
+  def testVoidSerdeConsumption(): Unit = {
```

Comment:
Do we really need a scala integration test for this? It seems like it should be covered well enough by the unit tests you wrote

Reply:
`testVoidSerdeConsumption` removed.